src/forms-opportunities/handler.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import RUMAPIClient from '@adobe/spacecat-shared-rum-api-client';
+import { getRUMDomainkey } from '../support/utils.js';
+import { AuditBuilder } from '../common/audit-builder.js';
+import { wwwUrlResolver } from '../common/audit.js';
+
+const DAILY_THRESHOLD = 200;
+const INTERVAL = 7; // days
+const FORMS_OPPTY_QUERIES = [
+  'cwv',
+  'form-vitals',
+];
+
+export async function formsAuditRunner(auditUrl, context, site) {
+  const rumAPIClient = RUMAPIClient.createFrom(context);
+  const domainkey = await getRUMDomainkey(site.getBaseURL(), context);
+  const options = {
+    domain: auditUrl,
+    domainkey,
+    interval: INTERVAL,
+    granularity: 'hourly',
+  };
+
+  const formsAuditLinks = await rumAPIClient.queryMulti(FORMS_OPPTY_QUERIES, options);
+  const cwvMap = new Map(
+    formsAuditLinks.cwv.map((cwv) => [cwv.url, cwv]),
+  );
+
+  const auditResult = {
+    formVitals: formsAuditLinks['form-vitals'].filter((data) => {
+      // Calculate the sum of all values inside the `pageview` object
+      // eslint-disable-next-line max-len
```

Comment:
pls avoid disabling eslint and refactor accordingly.

Reply:
done